profiles/marcokumu.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/90620737/181904436-f701dab8-5940-4572-951d-4c714cbca8ee.png",
+    "issueId": 1268,
+    "name": "Okumu Mark",
+    "username": "marcokumu"
```

Comment:
There seems to be a 4-space indentation. We use a 2-space indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/90620737/181904436-f701dab8-5940-4572-951d-4c714cbca8ee.png",
  "issueId": 1268,
  "name": "Okumu Mark",
  "username": "marcokumu"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix